packages/ds-medicare-gov/src/styles/components/_SimpleFooter.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable declaration-no-important -- Temporarily ignore until mgov can be refactored */
```

Comment:
Ignoring as it's `.m-c-` namespaced styles as they were added by outside teams and it's unknown what they're overwriting without doing a deeper dive.